source/_integrations/hue.markdown

```diff
@@ -102,3 +102,51 @@ The Hue API doesn't activate scenes directly; rather, they must be associated wi
 Neither group names nor scene names are guaranteed unique in Hue. If you are observing unexpected behavior from calling Hue scenes in Home Assistant, make the names of your Hue scenes more specific in the Hue app.
 
 The Hue hub has limited space for scenes and will delete scenes if new ones get created that would overflow that space. The API documentation says this is based on the scenes that are "least recently used."
+
+## Using Switches in Automations
+
+This integration will detect button presses from the various switch accessories in Hue for use in automations. In the example below, when the off button on the `living_room_switch` dimmer is pressed briefly, the `living_room_tv` media player will also be switched off.
+
+```yaml
+alias: TV - Turn off when light switch "off" is pressed
+trigger:
+  - platform: event
```

Comment:
Why not document the use of device automation instead?

Reply:
Updated